src/board_controller/brainalive/brainalive.cpp

```diff
@@ -371,21 +386,45 @@ void BrainAlive::adapter_1_on_scan_found (
 void BrainAlive::read_data (simpleble_uuid_t service, simpleble_uuid_t characteristic,
     uint8_t *data, size_t size, int channel_num)
 {
-    if (size != BRAINALIVE_PACKET_SIZE)
+    static int SOFTWARE_GAIN, HARDWARE_GAIN, REFFRENCE_VOLTAGE;
```

Comment:
there is no reason to make these static afaict, and we dont use capital case for variables